tsdb/index/postings.go

```diff
@@ -224,21 +224,27 @@ func (p *MemPostings) All() Postings {
 
 // EnsureOrder ensures that all postings lists are sorted. After it returns all further
 // calls to add and addFor will insert new IDs in a sorted manner.
-func (p *MemPostings) EnsureOrder() {
+// Parameter numberOfConcurrentProcesses is used to specify the maximal number of
+// CPU cores used for this operation. If it is <= 0, GOMAXPROCS(0) is used.
```

Comment:
```suggestion
// CPU cores used for this operation. If it is <= 0, GOMAXPROCS is used.
```

_This comment applies in other comment lines too._